css/index.css

```
@@ -0,0 +1,66 @@
+body {
+   background-color: #FBF2C4; 
+   color: #008585;
+   margin: 40px, 100, 120px, 80px;
```

```suggestion
   margin: 40px, 100px, 120px, 80px;
```
be consistent with units